recipes/applaunchservices/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "applaunchservices" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3da9d5435aec18eb70000f51cdf5ca428ba4bf03f36d922eb04635f8b5af4296
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
+    - pytest
+    - pyqt
+  imports:
+    - pytest
+    - applaunchservices
+    - PyQt5
```

Comment:
The only test necessary here is that `applaunchservices` imports. The other import's are not related to this package.

I understand that these packages are necessary to run the `applaunchservices` test suite, but the tests for conda packages are simpler. It's usually enough to test that the package imports correcly.